### PR TITLE
06 - Migrations!

### PR DESCRIPTION
**Pull request by mar7in91Wolox**
# Summary
- I configured the DB
- I made the InitialMigration
- I verified the table creation with pgAdmin

# Trello Card
https://trello.com/c/KyD80Ht6

# Screenshots

image

### CODE REVIEW

**Comment by mar7in91Wolox** [comment]
@truccomarcos Rebase done!

**Comment by truccomarcos** [comment]
@mar7in91Wolox Is it possible there's a DataBaseContextModelSnapshot file missing here?

**Comment by mar7in91Wolox** [comment]
@truccomarcos : 
 I fixed the missing snapshot file by dropping the existing data base and making a new Initial Migration.

File: training-net/src/Migrations/20180823131401_InitialMigration.cs [hunk 1]
```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+using Npgsql.EntityFrameworkCore.PostgreSQL.Metadata;
+
+namespace trainingnet.Migrations
```
Comment (truccomarcos) [inline training-net/src/Migrations/20180823131401_InitialMigration.cs]: Make sure this namepsaces are the same as the other ones (training_net)

File: training-net/src/Migrations/DataBaseContextModelSnapshot.cs [hunk 2]
```diff
@@ -0,0 +1,42 @@
+﻿// <auto-generated />
+using System;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Storage.ValueConversion;
+using Npgsql.EntityFrameworkCore.PostgreSQL.Metadata;
+using training_net.Repositories.Database;
+
+namespace trainingnet.Migrations
```
Comment (truccomarcos) [inline training-net/src/Migrations/DataBaseContextModelSnapshot.cs]: Same here

File: training-net/src/Migrations/20180823131401_InitialMigration.cs (hunk 1 above)
Comment (mar7in91Wolox) [reply training-net/src/Migrations/20180823131401_InitialMigration.cs]: Done!

File: training-net/src/Migrations/DataBaseContextModelSnapshot.cs (hunk 2 above)
Comment (mar7in91Wolox) [reply training-net/src/Migrations/DataBaseContextModelSnapshot.cs]: Done!

Comment (truccomarcos) [reply training-net/src/Migrations/DataBaseContextModelSnapshot.cs]: I think you forgot to change this one :stuck_out_tongue: (training_net)

**Pull request merged**